web/api/v1/errors_test.go

```diff
@@ -135,7 +135,8 @@ func createPrometheusAPI(q storage.SampleAndChunkQueryable) *route.Router {
 		nil,
 		nil,
 		false,
-		false,
+		config.RemoteWriteProtoMsgs{config.RemoteWriteProtoMsgV1, config.RemoteWriteProtoMsgV2},
+		false, // Disable experimental reduce remote write proto support.
```

Comment:
is the last parameter here still allowed?

Reply:
Hm, weird. This flag is for OTLP receive, plus it looks fine to have both supported in this test case (entry above?). Feels we can remove comment now.

Reply:
I think months ago we had our own additional bool flag for whether or not to use the new `reduced` format, before the otlp support was merged into main